assets/erc-7582/MADVAccount.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: AGPL-3.0-only
```

Comment:
Unfortunately we cannot use GPL or other copyleft code as reference implementations. Doing so would mean anyone who looks at the reference implementation would also have to release their code as GPL.

Personally, I love the GPL, but it doesn't make sense in a standards repository.

Reply:
I have removed the license identifier 👍